src/frontend/components/UI/Dialog/components/Dialog.tsx

```diff
@@ -19,100 +23,73 @@ interface DialogProps {
   onClose: () => void
 }
 
+const StyledPaper = styled(Paper)(() => ({
+  backgroundColor: 'var(--modal-background)',
+  color: 'var(--text-default)',
+  maxWidth: '100%',
+  '&:has(.settingsDialogContent)': {
+    height: '80%'
+  }
+}))
+
 export const Dialog: React.FC<DialogProps> = ({
   children,
   className,
   showCloseButton = false,
   onClose
 }) => {
-  const dialogRef = useRef<HTMLDialogElement | null>(null)
-  const onCloseRef = useRef(onClose)
-  onCloseRef.current = onClose
-  const [focusOnClose, setFocusOnClose] = useState<HTMLElement | null>(null)
+  const [open, setOpen] = useState(true)
   const { disableDialogBackdropClose } = useContext(ContextProvider)
 
   useEffect(() => {
-    setFocusOnClose(document.querySelector<HTMLElement>('*:focus'))
+    // HACK: Focussing the dialog using JS does not seem to work
+    //       Instead, simulate one or two tab presses
+    // One tab to focus the dialog
+    window.api.gamepadAction({ action: 'tab' })
+    // Second tab to skip the close button if it's shown
+    if (showCloseButton) window.api.gamepadAction({ action: 'tab' })
   }, [])
 
-  const close = () => {
-    onCloseRef.current()
-    if (focusOnClose) {
-      setTimeout(() => focusOnClose.focus(), 200)
-    }
-  }
-
-  useEffect(() => {
-    const dialog = dialogRef.current
-    if (dialog) {
-      const cancel = () => {
-        close()
-      }
-      dialog.addEventListener('cancel', cancel)
-
-      if (disableDialogBackdropClose) {
```

Comment:
this was here to support the Accessibility option to disable the `close dialog when clicking outside` behavior. In this PR that's not working

Reply:
Whoops, somehow understood this as "Disable Esc key closing dialogs"
Fixed!